permissions/SharedStorage.permission

```diff
@@ -0,0 +1,11 @@
+# -*- mode: sh -*-
+
+# x-sailjail-translation-catalog = sailjail-permissions
+# x-sailjail-translation-key-description = permission-la-sharedstorage
+# x-sailjail-description = Shared storage
+# x-sailjail-translation-key-long-description = permission-la-sharedstorage_description
+# x-sailjail-long-description = Access Shared storage
```

Comment:
I think this should be a bit more verbose. Like "Access application storage shared between users". Also add the permission to README.md please.

Reply:
I will keep this wording for a now and think about another.